### PR TITLE
skipper: update pod-deletion-cost-controller logging configuration

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
* log to infra account
* save last chunk of logs into container status on error

### CODE REVIEW

**Comment by MustafaSaber** [comment]
👍

**Comment by szuecs** [comment]
Why should we log into infra account?

File: cluster/manifests/skipper/pod-deletion-cost-controller.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ spec:
         application: skipper-ingress
         component: pod-deletion-cost-controller
         deployment: pod-deletion-cost-controller
+      annotations:
+        logging/destination: "{{ .Cluster.ConfigItems.log_destination_infra }}"
```
Comment (szuecs) [inline cluster/manifests/skipper/pod-deletion-cost-controller.yaml]: why?

Comment (AlexanderYastrebov) [reply cluster/manifests/skipper/pod-deletion-cost-controller.yaml]: We have discussed and agreed to log into both cluster and infra accounts.

**Comment by AlexanderYastrebov** [comment]
Closing in favour of #8186

File: cluster/manifests/skipper/pod-deletion-cost-controller.yaml (hunk 1 above)
Comment (AlexanderYastrebov) [reply cluster/manifests/skipper/pod-deletion-cost-controller.yaml]: See #8186

**Pull request closed without merging**